doc/ch-image_desc.rst

```diff
@@ -239,6 +248,13 @@ Options:
     Parse :code:`IMAGE_REF`, print a parse report, and exit successfully
     without talking to the internet or touching the storage directory.
 
+This script tries to target the appropriate architecture when pulling. For
+example, when pulling the image tagged :code:`centos:7`, first check for the
+existence of a fat manifest. If found, check for the architecture matching the
+host. If a match is found, pull it; otherwise proceed as if the fat manifest
+does not exist. If a fat manifest does not exist, pull the image without trying
+to guess the appropriate architecture.
```

Comment:
E.g. I don't know what this means. What architecture do I get? This should explain what the results are, not what's happening under the hood.